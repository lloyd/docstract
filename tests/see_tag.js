/**
 * The see tag lets you reference other things.
 * You can pass whatever text you want to the see tag.
 *
 * @see #module/file/read
 *
 * @desc in the case above, we're passing something that
 * looks like a hashtag.
 *
 * @see #methodName
 * @seealso #ClassName
 * @see_also ClassName#methodName
 *
 * @desc
 * Those three examples use the forms that jsdoc likes to
 * eat. But really, it's all up to your rendering code.
 * So go ahead, make a convention, it's fun!
 */

/**
 * @func MyFunc
 * Now functions too can have see tags.  They can occur
 * in almost any type of documentation block.
 * @see DoYouSeeNow?
 * @see HowAboutNow?
 */
