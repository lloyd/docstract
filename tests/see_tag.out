{
  "desc": "The see tag lets you reference other things.\nYou can pass whatever text you want to the see tag.\n\nin the case above, we're passing something that\nlooks like a hashtag.\n\nThose three examples use the forms that jsdoc likes to\neat. But really, it's all up to your rendering code.\nSo go ahead, make a convention, it's fun!", 
  "filename": "tests/see_tag.js", 
  "functions": [
    {
      "desc": "Now functions too can have see tags.  They can occur\nin almost any type of documentation block.", 
      "name": "MyFunc", 
      "see": [
        "DoYouSeeNow?", 
        "HowAboutNow?"
      ], 
      "source_lines": [
        20, 
        26
      ]
    }
  ], 
  "module": "see_tag", 
  "see": [
    "#module/file/read", 
    "#methodName", 
    "#ClassName", 
    "ClassName#methodName"
  ], 
  "source_lines": [
    1, 
    19
  ]
}
